src/main/java/seedu/address/storage/AddressBookStorage.java

```diff
@@ -8,38 +8,36 @@
 import seedu.address.model.ReadOnlyAddressBook;
 
 /**
- * Represents a storage for {@link seedu.address.model.AddressBook}.
- */
+* Represents a storage for {@link seedu.address.model.AddressBook}.
+*/
 public interface AddressBookStorage {
-
     /**
-     * Returns the file path of the data file.
-     */
+    * Returns the file path of the data file.
+    */
     Path getAddressBookFilePath();
 
     /**
-     * Returns AddressBook data as a {@link ReadOnlyAddressBook}.
-     *   Returns {@code Optional.empty()} if storage file is not found.
-     * @throws DataConversionException if the data in storage is not in the expected format.
-     * @throws IOException if there was any problem when reading from the storage.
-     */
+    * Returns AddressBook data as a {@link ReadOnlyAddressBook}.
+    *   Returns {@code Optional.empty()} if storage file is not found.
+    * @throws DataConversionException if the data in storage is not in the expected format.
+    * @throws IOException if there was any problem when reading from the storage.
+    */
     Optional<ReadOnlyAddressBook> readAddressBook() throws DataConversionException, IOException;
 
     /**
-     * @see #getAddressBookFilePath()
-     */
+    * @see #getAddressBookFilePath()
+    */
     Optional<ReadOnlyAddressBook> readAddressBook(Path filePath) throws DataConversionException, IOException;
 
     /**
-     * Saves the given {@link ReadOnlyAddressBook} to the storage.
-     * @param addressBook cannot be null.
-     * @throws IOException if there was any problem writing to the file.
-     */
+    * Saves the given {@link ReadOnlyAddressBook} to the storage.
+    * @param addressBook cannot be null.
+    * @throws IOException if there was any problem writing to the file.
+    */
     void saveAddressBook(ReadOnlyAddressBook addressBook) throws IOException;
 
     /**
-     * @see #saveAddressBook(ReadOnlyAddressBook)
-     */
+    * @see #saveAddressBook(ReadOnlyAddressBook)
```

Comment:
I'm surprised these passes checkStyle o.o

Reply:
please ignore, they are just fixes by prettier and yea checkstyle doesn't see them